config/config_test.go

```diff
@@ -366,7 +366,6 @@ func TestDecodeConfig_Invalid(t *testing.T) {
 		{name: "pipeline-must-have-receiver", expected: errPipelineMustHaveReceiver},
 		{name: "pipeline-exporter-not-exists", expected: errPipelineExporterNotExists},
 		{name: "pipeline-processor-not-exists", expected: errPipelineProcessorNotExists},
-		{name: "pipeline-must-have-processors", expected: errPipelineMustHaveProcessors},
```

Comment:
This test should be replaced with "pipeline-with-zero-processors" that should give no error.